### ISSUE TITLE
Is Carbon still being maintained?

### ISSUE THREAD

**Issue by MaxGiting** [briannesbitt/Carbon#654]
Nothing has been merged or committed so far this year. Are there any plans to continue Carbons support and development @briannesbitt ?

**Comment by briannesbitt** [comment]
Yes... but currently the usual life events are getting in the way though.

**Comment by MaxGiting** [comment]
Brilliant. Thank you for the swift reply!

**Comment by JayBizzle** [comment]
Perhaps @lucasmichot could be added as a maintainer. Just throwing it out there 💣 😆

**Comment by lucasmichot** [comment]
I am OK with that, up to @briannesbitt
